baseline/OSPS-GV.yaml

```
@@ -12,8 +12,17 @@ criteria:
       The project documentation MUST include the
       Roles and Responsibilities for members of the
       project.
-    rationale: # TODO
-    implementation: # TODO
+    rationale: |
+      Documenting project roles and responsibilities
+      helps project particpants, potential contributors,
+      and downstream consumers have an accurate
+      understand of who is working on the project
+      and what areas or authority they may have.
+    implementation: |
+      Documenting project particpants and their roles
```

```suggestion
      Document project participants and their roles
```
Not sure what our to-be-written style guide will say on this matter, but "Document" seems better than "Documenting" because it's imperative. And either way, the typo fix is good. :-)